### DISCUSSION TITLE
Ukrainian Language Localization

### DISCUSSION THREAD

**Discussion by dolgachio** [rollup/rollup#5096] [Ideas]
Hi, Rollup team!

You already have the Chinese localization for the documentation, which is so cool!

I am from Ukraine, and I was thinking, if I come up with something similar to [https://github.com/rollup/rollup-docs-cn](https://github.com/rollup/rollup-docs-cn) but translated into the Ukrainian language, would you consider using it in the [https://rollupjs.org/](https://rollupjs.org/)?

Thank you for a such cool project!

**Comment by lukastaegert** [comment]
Definitely! I think rollup-docs-cn already has a nice setup that makes it easy to keep translations up to date that you can copy. Once you have something reasonably far evolved, we can move it into the Rollup organization, set up deployment and link it from the main website.

**Comment by IgorMonster** [comment]
Its great idea !
Ukrainian language is realy bored !
Слава Україні !
